notification-server/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "notification-server"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I suggest to add the author, so people (even after deleting the git history) have another way to figure out the origin/author of the code.

authors = ["Concordium AG <developers@concordium.com>"]